man/satellite-clone.1

```diff
@@ -0,0 +1,82 @@
+.\" Automatically generated by Pandoc 1.17.0.3
+.\"
+.TH "SATELLITE\-CLONE" "1" "" "Version 1.2" "satellite\-clone documentation"
+.hy
+.SH NAME
+.PP
+\f[B]satellite\-clone\f[] \[em] Setup a Satellite 6.1 or 6.2 install
+with restored backup data.
+.SH SYNOPSIS
+.PP
+\f[B]satellite\-clone\f[] [\f[B]\-h\f[]|\f[B]\-\-help\f[]]
+[\f[B]\-v\f[]|\f[B]\-\-verbose\f[]] [\f[B]\-\-start\-at\-task TASK\f[]]
+[\f[B]\-\-step\f[]]
+.SH DESCRIPTION
+.PP
+Restores a working Satellite 6 instance from a backup.
+Currently satellite 6.1 and 6.2 backups are supported.
+.SS You will need:
+.IP \[bu] 2
+A blank, clean install, RHEL 7 machine,
+.IP \[bu] 2
+A satellite backup (katello\-backup in 6.1 and 6.2), see the required
+files below
+.RS 2
+.IP \[bu] 2
+Standard backup scenario: config_files.tar.gz, mongo_data.tar.gz,
+pgsql_data.tar.gz, (optional) pulp_data.tar
+.IP \[bu] 2
+Online backup or RHEL 6 to 7 migration scenario: config_files.tar.gz,
+mongo_dump folder, foreman.dump, candlepin.dump, (optional)
+pulp_data.tar
+.RE
+.SS To perform the clone:
+.IP \[bu] 2
+Place the backup files in a folder on the blank machine.
+This folder path is specified in the "backup_dir" variable in
+/etc/satellite\-clone/satellite\-clone\-vars.yml and can be changed.
+.IP \[bu] 2
+If you are cloning RHEL 6 backup data to a RHEL 7 machine, update the
+variable "rhel_migration" (Satellite 6.2 only) to true in
+satellite\-clone\-vars.yml.
+.IP \[bu] 2
+In the same file either add an activation key from the Red Hat portal OR
+setup the proper Satellite subscriptions yourself and change
+"register_to_portal" to false in satellite\-clone\-vars.yml
+.IP \[bu] 2
+Run satellite\-clone
+.PP
+Backups from a Satellite 6.2 instance on RHEL 6 can be restored on a
+RHEL 7 machine.
+To do this, change the "rhel_migration" variable to true in
+/etc/satellite\-clone/satellite\-clone\-vars.yml
+.SS Options
+.TP
+.B \-v, \-\-verbose
+Verbose output
+.RS
+.RE
+.TP
+.B \-\-start\-at\-task TASK
```

Comment:
Curious, how does the user know what tasks are available?  Would it be worthwhile to include an example or is obvious when based on past runs of the tool?